docs-chef-io/layouts/shortcodes/chef-workstation/ctl_chef_install.md

```diff
@@ -4,4 +4,19 @@ then emit a `Policyfile.lock.json` file that describes the locked policy
 set. The `Policyfile.lock.json` file may be used to install the locked
 policy set to other machines and may be pushed to a policy group on the
 Chef Infra Server to apply that policy to a group of nodes that are
-under management by Chef.
\ No newline at end of file
+under management by Chef.
+
+<div class="admonition-note">
+
+<p class="admonition-note-title">Note</p>
+
+<div class="admonition-note-text">
+
+By default, the cookbook cache is located in `~/.chef-workstation`
+on macOS / Linux and in `%LOCALAPPDATA%\chef-workstation` on Windows.
+This can be changed by defining the `CHEF_WORKSTATION_HOME` environment
+variable and setting its value to the desired cache directory.
```

Comment:
```suggestion
By default, the cookbook cache is located in `~/.chef-workstation`
on macOS and Linux, and in `%LOCALAPPDATA%\chef-workstation` on Windows.
You can change the cache location by defining the `CHEF_WORKSTATION_HOME` environment
variable and setting its value to the desired cache directory.
```

Reply:
Do we have documentation somewhere for setting the `CHEF_WORKSTATION_HOME` environment
variable? I'm not finding anything. We should have something somewhere telling people how to do this first.

Reply:
No existing documentation. This is the intent of the note here, this environment variable primarily caters to cache used by `chef-cli`.

Reply:
I can add a separate note to help users set an environment variable in general.

Reply:
Is `CHEF_WORKSTATION_HOME` just set in a bashrc or zshrc file? If so we could probably just do something like:

> Set the desired location of the cache directory by setting the `CHEF_WORKSTATION_HOME` environment variable in your `.bashrc` or `zshrc` file. For example, `CHEF_WORKSTATION_HOME="~/.workstation"`.

Reply:
yes, that is apt in macOS and Linux, on Windows the technique is based on adding via settings or from terminal by using [setx](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/setx). i'll modify and push a commit.